browser/help/topics/v4-hts.md

```diff
@@ -66,11 +66,549 @@ The above example will retrieve the entire frequency struct for each variant. To
+    - `age_hist_het`: Histogram for age in all heterozygous release samples calculated on high quality genotypes.
+      - `bin_edges`: Bin edges for the age histogram.
+      - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+      - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+      - `n_larger`: Count of age values falling above highest histogram bin edge.
+    - `age_hist_hom`: Histogram for age in all homozygous release samples calculated on high quality genotypes. If variant is in the pseudoautosomal regions of chrX or chrY, this histogram also includes age counts of hemizygous samples.
+      - `bin_edges`: Bin edges for the age histogram.
+      - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+      - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+      - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `joint`: Struct of combined (joint) exomes and genomes allele frequency information.
+  - `freq`: Array of allele frequency information (AC, AN, AF, homozygote count) across the combined (joint) gnomAD exomes and genomes and for each genetic ancestry group.
+    - AC: Combined (exomes + genomes) alternate allele count in release.
+    - AF: Combined (exomes + genomes) alternate allele frequency, (AC/AN), in release.
+    - AN: Total number of alleles across exomes and genomes in release.
+    - homozygote_count: Count of homozygous alternate individuals across exomes and genomes in release.
+  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency across both exomes and genomes. Excludes Amish (ami), Ashkenazi Jewish (asj), European Finnish (fin), and "Remaining individuals" (remaining) groups.
+    - `AC`: Alternate allele count in the group with the maximum allele frequency.
+    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
+    - `AN`: Total number of alleles in the group with the maximum allele frequency.
+    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
+    - `gen_anc`: Genetic ancestry group with maximum allele frequency.
+  - `faf`: Array of combined exomes and genomes filtering allele frequency information (AC, AN, AF, homozygote count). Note that the values in array will correspond to the joint or combined value if the variant had a defined filtering allele frequency in both data types, otherwise this array will contain filtering allele frequencies only for the data type associated with the Hail Table (in this case, exomes).
+    - `faf95`: Combined exomes and genomes filtering allele frequency (using Poisson 95% CI).
+    - `faf99`: Combined exomes and genomes filtering allele frequency (using Poisson 99% CI).
```

Comment:
```suggestion
  - `faf`: Array of combined exomes and genomes filtering allele frequency information (AC, AN, AF, homozygote count). Note that the values in array will correspond to the joint or combined value if the variant had a defined filtering allele frequency in both data types, otherwise this array will contain filtering allele frequencies only for the data type associated with the Hail Table (in this case, exomes).
    - `faf95`: Combined exomes and genomes filtering allele frequency (using Poisson 95% CI).
    - `faf99`: Combined exomes and genomes filtering allele frequency (using Poisson 99% CI).
  - `fafmax`: Information about the genetic ancestry group with the maximum filtering allele frequency.
    - `faf95_max`: Maximum filtering allele frequency (using Poisson 95% CI).
    - `faf95_max_gen_anc`: Genetic ancestry group with the maximum filtering allele frequency (95% CI).
    - `faf99_max`: Maximum filtering allele frequency (using Poisson 99% CI).
    - `faf99_max_gen_anc`: Genetic ancestry group with the maximum filtering allele frequency (99% CI).
  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency across both exomes and genomes. Excludes Amish (ami), Ashkenazi Jewish (asj), European Finnish (fin), and "Remaining individuals" (remaining) groups.
    - `AC`: Alternate allele count in the group with the maximum allele frequency.
    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
    - `AN`: Total number of alleles in the group with the maximum allele frequency.
    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
    - `gen_anc`: Genetic ancestry group with maximum allele frequency.
```